docs/design/datacontracts/data_descriptor.md

```diff
@@ -92,6 +93,15 @@ The value must be an integral constant within the range of its type.  Signed val
 natural encoding.  Pointer values need not be aligned and need not point to addressable target
 memory.
 
+## Sub-descriptor descriptors
+
+Each sub-descriptor descriptor is effectively a global with a type of `pointer`. They will consist of:
+* a name
+* a pointer value
+
+If these values are non-null, the pointer represents another JSON data descriptor with the specification described in this document.
```

Comment:
```suggestion
If the value is non-null, the pointer points to another [contract descriptor](contract-descriptor.md#contract-descriptor-1).
```
I assume it will point to `DotNetRuntimeContractDescriptor` that speced in the other doc, not directly to JSON.

Reply:
Also, we use `DotNetRuntimeContractDescriptor` for both the main contract export and the data structure that it points to. We may want to rename the data structure (e.g. to just `ContractDescriptor`) to avoid confusion now that there will be multiple instances of it in the system.